### PR TITLE
Feature/automated testing updated xUnit traits

### PR DESCRIPTION
**Pull request by markusstuhler**

### CODE REVIEW

File: e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ namespace IIoTPlatform_E2E_Tests.Standalone {
     /// </summary>
     [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
     [Collection("IIoT Platform Test Collection")]
+    [Trait("PublisherMode", "standalone")]
```
Comment (koepalex) [inline e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs:20]: can you add both name and values to the TestConstants?

File: e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@ namespace IIoTPlatform_E2E_Tests.Orchestrated
     /// </summary>
     [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
     [Collection("IIoT Platform Test Collection")]
-    [Trait("IsApiTest", "true")]
+    [Trait("PublisherMode", "orchestrated")]
```
Comment (koepalex) [inline e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs:23]: can you add both name and values to the TestConstants?

**Review by koepalex — APPROVED** [review]
:shipit:

File: e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs (hunk 2 above)
Comment (karok2m) [reply e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs:23]: Could enum work better here ?

File: e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs (hunk 1 above)
Comment (markusstuhler) [reply e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs]: Done.

File: e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs (hunk 2 above)
Comment (markusstuhler) [reply e2e-tests/IIoTPlatform-E2E-Tests/Orchestrated/PublishSingleNodesTestTheory.cs]: Done.

File: tools/e2etesting/pipeline.yml [hunk 3]
```diff
@@ -53,7 +53,7 @@ stages:
     steps:
     - template: steps/runtests.yml
       parameters:
-        RunApiTests: true
```
Comment (karok2m) [inline tools/e2etesting/pipeline.yml:56]: On a side note, shall we do some renaming of pipeline YAML files to be more explicit ?
* `pipeline.yml` -> `pipeline_orchestrated.yml` 
* `pipelinestandalone.yml` -> `pipeline_standalone.yml`

Comment (koepalex) [reply tools/e2etesting/pipeline.yml:56]: Good Idea, we should do it in another PR

**Pull request merged**